src/renderers/webgpu/nodes/WGSLNodeBuilder.js

```diff
@@ -219,7 +219,14 @@ class WGSLNodeBuilder extends NodeBuilder {
 	 */
 	needsToWorkingColorSpace( texture ) {
 
-		return texture.isVideoTexture === true && texture.colorSpace !== NoColorSpace;
```

Comment:
I think this bit isn't 100%. `needsToWorkingColorSpace()` was implemented as a workaround for video textures. Since they are now treated as all other textures in WebGPU, we can simply remove the method and use the one from `NodeBuilder` (which always returns `false`).

Reply:
The examples looks good to me after the removing this overwrite. Do I miss something on my side?

Reply:
Thanks for the fix.